LevelComponents/EntitySet.cpp

```diff
@@ -162,6 +163,7 @@ namespace LevelComponents
             if (offset > 15)
             {
                 // TODO: deal with exception
+                std::cout << "load entityset error: too many palettes" << std::endl;
```

Comment:
No, not this kind of log. Use the widget console so that users can see the warning message (and remove the `<iostream>`)